cms/utils/copy_plugins.py

```diff
@@ -29,7 +29,7 @@ def copy_plugins_to(old_plugins, to_placeholder,
         for idx, plugin in enumerate(new_plugins):
             if plugin.parent_id is None:
                 plugin.parent_id = parent_plugin_id
-                plugin.save()
+                plugin.save(update_fields=['parent'])
```

Comment:
Since the significant of this will only be clear to those familiar with djangocms-text-ckeditor, let's add a comment so someone doesn't remove this later.